accounts/usbwallet/wallet.go

```diff
@@ -479,7 +479,7 @@ func (w *wallet) Derive(path accounts.DerivationPath, pin bool) (accounts.Accoun
 
 	if _, ok := w.paths[address]; !ok {
 		w.accounts = append(w.accounts, account)
-		w.paths[address] = path
+		copy(w.paths[address], path)
```

Comment:
This doesn't make a new copy. It retrieves `w.paths[address]`, which will be an empty slice (nil) and then copy into it, limited to the length of the smallest operand (i.e. `nil`, length of 0`) and return. I.e. this is a noop.

Reply:
yeah I just saw debugging the other PR :disappointed:

Reply:
Alternatively, we can make a `Copy` method for `accounts.DerivationPath`, might come in handy long term for Clef too.